### DISCUSSION TITLE
[mini.git] Feature request: Show signs in gutter

### DISCUSSION THREAD

**Discussion by ckafi** [nvim-mini/mini.nvim#1283] [Ideas]
First of all: Thank you for your awesome work. I've been slowly replacing almost all my plugins with corresponding mini.* alternatives, and it's been going great. :heart: 

Now for my request: I would like to replace gitsigns.nvim with mini.git, but one missing feature has been holding me back: the gutter signs for added, modified, and removed lines. If you would implement these, I would switch in a heart beat.

**Comment by echasnovski** [comment]
Thanks for the kind words!

Well, today is a lucky day. The ['mini.diff'](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-diff.md) module was created specifically as a 'gitsigns.nvim' replacement in my config. I am using 'mini.diff' daily and I am not missing anything from 'gitsigns.nvim'.

**Comment by ckafi** [comment]
Oh, I was somehow under the impression that mini.diff was for diff mode. Thank for the info!
